src/Symfony/Component/PropertyInfo/PropertyAttributesExtractorInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+/**
+ * @author Andrew Alyamovsky <andrew.alyamovsky@gmail.com>
+ */
+interface PropertyAttributesExtractorInterface
```

Comment:
```suggestion
interface PropertyAttributeExtractorInterface
```
To be consistent with other extractors. For example, even if `getTypes` is plural, `PropertyTypeExtractorInterface` is singular.